### ISSUE TITLE
Tem como separar cada boilerplate em 1 repositório diferente?

### ISSUE THREAD

**Issue by suissa** [vitorbritto/boilerplates#18]
Seria muito mais fácil para clonar e iniciar apenas o necessário.

**Comment by vitorbritto** [comment]
Sim, futuramente será uma necessidade. Vou aperfeiçoar a cli para tratar melhor estes repositórios também. Fiz um _update_ hoje, mas para categorizar melhor apenas.
 
Assim que migrar os _boilerplates_ para as novas tecnologias e ferramentas (Babel, PostCSS, JSPM, etc), farei isso. =]
